lib/ProofSet.js

```diff
@@ -98,37 +81,16 @@ module.exports = class ProofSet {
     // create the new proof (suites MUST output a proof using the security-v2
     // `@context`)
     const proof = await suite.createProof({
-      document: input, purpose, documentLoader,
-      expansionMap, compactProof});
+      document: input, purpose, documentLoader, expansionMap
+    });
 
-    if(compactProof) {
-      // compact proof to match document's context
-      const expandedProof = {
-        [constants.SECURITY_PROOF_URL]: {'@graph': proof}
-      };
-      // account for type-scoped `proof` definition by getting document types
-      const {types, alias} = await _getTypeInfo(
-        {document, documentLoader, expansionMap});
-      expandedProof['@type'] = types;
-      const ctx = jsonld.getValues(document, '@context');
-      const compactProof = await jsonld.compact(
-        expandedProof, ctx,
-        {documentLoader, expansionMap, compactToRelative: false});
-      delete compactProof[alias];
-      delete compactProof['@context'];
-
-      // add proof to document
-      const key = Object.keys(compactProof)[0];
-      jsonld.addValue(document, key, compactProof[key]);
-    } else {
-      // in-place restore any existing proofs
-      /*if(existingProofs) {
-        document[proofProperty] = existingProofs;
-      }*/
-      // add new proof
-      delete proof['@context'];
-      jsonld.addValue(document, proofProperty, proof);
-    }
+    // in-place restore any existing proofs
+    /*if(existingProofs) {
+      document[proofProperty] = existingProofs;
+    }*/
+    // add new proof
+    delete proof['@context'];
```

Comment:
I think we can remove all of these lines (the commented code and the code deleting the context from the proof).

Reply:
Deleted.